### PR TITLE
Reworked linux ss page

### LINKED ISSUE #687: Pages with multiple commands per example
Related to tldr-pages/tldr-node-client#64 and somewhat to #685. 

The current linter at `tldr-pages/tldr-lint#multiple-examples` (npm install path) detects multiple commands per example. It also comes with a tabular output formatter. That enabled me to do the following: 

`tldrl pages/ -t 2>&1 | cut -f1 | uniq | grep "pages.*" -o`

Resulting in this beauty.
```
pages/common/cal.md
pages/common/ls.md
pages/common/redis-cli.md
pages/common/rename.md
pages/common/salt-key.md
pages/common/scp.md
pages/common/srm.md
pages/common/touch.md
pages/common/which.md
pages/common/zpool.md
pages/linux/md5sum.md
pages/linux/sha1sum.md
pages/linux/sha224sum.md
pages/linux/sha256sum.md
pages/linux/sha384sum.md
pages/linux/sha512sum.md
pages/linux/ss.md
pages/linux/wpa_cli.md
pages/osx/md5.md
pages/osx/sysctl.md
pages/osx/systemsetup.md
pages/osx/xed.md
```

These are all files with multiple commands per example. 

EDIT: @notpeter had a go at them, we're almost through.

- [x] pages/common/scp.md
- [x] pages/common/zpool.md
- [x] pages/linux/ss.md
- [x] pages/linux/wpa_cli.md

### PR DESCRIPTION
**Pull request by rubenvereecken**
An effort to close #687. This page was in a pretty bad condition. On the bright side, I learned a whole new command today.

I usually veer towards practical examples. This command especially calls for them because it has its own query language. Like tcpdump has one, kinda similar. I went out of my way and tried to come up with a few practical examples but my so-called expertise is limited to networks, I don't know much about Unix sockets so haven't managed to dredge up an example on those. 

This is somewhat more practically oriented than the average page. Lay it on me but be gentle, I didn't want to rewrite aaaall of the page just yet. Even though I nearly did.

### CODE REVIEW

File: pages/linux/ss.md [hunk 1]
```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
```
Comment (waldyrious) [inline pages/linux/ss.md]: We need to decide if we are to allow this, what character to use (`/`, `!`, `,`), and where this mechanism is admissible. In terms of formatting, I'd suggest we adhere to the [docopt](http://docopt.org/) standard as much as possible, i.e. use `|` for separating mutually exclusive options.

File: pages/linux/ss.md [hunk 2]
```diff
@@ -2,26 +2,26 @@
...
-- Show process(es) that using socket:
-
-`ss -p`
-
 - Filter TCP sockets by states, only/exclude:
 
 `ss {{state/exclude}} {{bucket/big/connected/synchronized/...}}`
 
-- Filter sockets by address and/or port:
+- Show all TCP sockets connected to the local HTTPS port:
```
Comment (waldyrious) [inline pages/linux/ss.md]: I'd suggest mentioning 443 in the description, e.g. "to the local HTTPS port (443):", to make it explicit.

File: pages/linux/ss.md [hunk 3]
```diff
@@ -2,26 +2,26 @@
...
+`ss -pt dst :{{ssh}}`
+
+- Show all UDP sockets connected on a local port within a range:
 
-`ss -t dst 1.2.3.4:80`
-`ss -u src 127/8`
-`ss -t 'dport >= :1024'`
-`ss -x "src /tmp/.X11-unix/*"`
-`ss -t state established '( dport = :ssh or sport = :ssh )'`
+`ss -u 'sport >= :{{lowest_port}} and sport <= :{{highest_port}}'`
```
Comment (waldyrious) [inline pages/linux/ss.md]: what does the `s` in `sport` stand for? We should indicate that in the description if possible.

Comment (rubenvereecken) [inline pages/linux/ss.md]: I'll change this example to use both sport and dport,  should be very intuitive that way

File: pages/linux/ss.md (hunk 2 above)
Comment (rubenvereecken) [inline pages/linux/ss.md]: :+1:

File: pages/linux/ss.md (hunk 1 above)
Comment (rubenvereecken) [inline pages/linux/ss.md]: This whole example bothers me tbh. I'd have preferred to describe 3 or so of these in other examples that also introduce filters, but I can't come up with a good Unix example so all I can manage is tcp and udp

Comment (waldyrious) [inline pages/linux/ss.md]: I don't think we should outright ban this construction; in cases like this where the commands are identical except for the option, it seems justifiable. But we need to agree that it's acceptable and when, so we don't impose restrictions on others that we don't follow universally.

File: pages/linux/ss.md (hunk 3 above)
Comment (waldyrious) [inline pages/linux/ss.md]: Good call :+1:

File: pages/linux/ss.md (hunk 1 above)
Comment (rubenvereecken) [inline pages/linux/ss.md]: The ipv4/6 example I originally fixed by having `ss -{{4|6}}`. The dash can be inside the braces even, but I'd insist on having `|` for separators we've had a couple of times.

Comment (waldyrious) [inline pages/linux/ss.md]: I'm not sure I understand your wording, but it seems we are in agreement. Including the dash next to each option may be more intuitive, though.

**Comment by rubenvereecken** [comment]
@waldyrious I reworked that first example. Next to a discussion about a convention, would this be good to go?

**Comment by waldyrious** [comment]
@rubenvereecken yes, but as I said in the inline thread: "Including the dash next to each option [i.e. inside the `{{}}`] may be more intuitive". Not a dealbreaker, so if you prefer the way it is, I'm ok with merging.

**Comment by rubenvereecken** [comment]
I've been in doubt about this myself. I'll change to the more intuitive.

**Pull request merged**